cmd/antrea-agent/options.go

```diff
@@ -254,9 +258,12 @@ func (o *Options) validateFlowExporterConfig() error {
 				return fmt.Errorf("IdleFlowExportTimeout is not provided in right format")
 			}
 			if o.idleFlowTimeout < o.pollInterval {
-				o.activeFlowTimeout = o.pollInterval
+				o.idleFlowTimeout = o.pollInterval
 				klog.Warningf("IdleFlowExportTimeout must be greater than or equal to FlowPollInterval")
 			}
+			if o.idleFlowTimeout > connections.StaleConnectionTimeout {
+				connections.StaleConnectionTimeout = 2 * o.idleFlowTimeout
+			}
```

Comment:
there is a bit of an asymmetry here: if `idleFlowTimeout` is 3m and `activeFlowTimeout` is 2m, the `StaleConnectionTimeout` will be set to `4m`. A symmetric solution would set it to `2*max(idleFlowTimeout, activeFlowTimeout)`. or 5 minutes, whichever is larger.

Reply:
Yes, I saw that. I assumed `idleFlowTimeout` to be always smaller than `activeFlowTimeout`, but I see that we do not have any check in the config options to make sure of that.

Will have a combined check to resolve that.